src/Framework/Microsoft.Build.Framework.csproj

```diff
@@ -174,5 +174,11 @@
   <ItemGroup>
     <Folder Include="Properties\" />
   </ItemGroup>
+  <ItemGroup>
+    <FilesToSign Include="$(OutDir)\$(AssemblyName)$(TargetExt)">
+      <Authenticode>Microsoft</Authenticode>
+      <StrongName>StrongName</StrongName>
+    </FilesToSign>
+  </ItemGroup>
```

Comment:
I made a different change to enable this functionality in xplat: https://github.com/Microsoft/msbuild/commit/899b5839c91b90900fc5e4e29a5f2d00cfb93aef

So the change made here and to the other `.csproj` can be left out of the merge if that's easy.  Otherwise we can remove it in a 2nd commit.